Tempo/Models/User.swift

```diff
@@ -112,6 +112,10 @@ class User: NSObject, NSCoding {
 		aCoder.encode(updatedAt, forKey: "updated_at")
 		aCoder.encode(username, forKey: "username")
 	}
+	
+	func equals(other: User) -> Bool {
```

Comment:
doesn't each user have a unique identifier, isn't that a better than using username?